TestAssets/TestProjects/FSharpTestAppSimple/TestAppSimple.fsproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
the app + netstandard2.0 lib is in PR https://github.com/dotnet/cli/pull/6914 (should show the error)
I'll add also 1.6

`dotnet/cli` test suite DSL is really nice for scripting workflow (because cli is awesome to automate dotnet projects!)